e2e/fabricaCommands.test.ts

```diff
@@ -43,8 +43,7 @@ describe("use", () => {
 
     // Then
     expect(commandResult).toEqual(TestCommands.success());
-    expect(commandResult.output).toContain("0.1.0-unstable");
-    expect(commandResult.output).toContain(`${currentFabricaVersion} <== current`);
+    expect(commandResult.output).toContain("0.0.1");
```

Comment:
Może późno jest, ale tego nie łapię. Czemu contain `0.0.1`?

Reply:
w output w tej chwili będą wszystkie dostępne wersje, czyli `0.0.1` i `0.1.0-unstable`. Po release będzie `0.0.1`, `0.1.0` i `0.2.0-unstable`. Czyli w obu przypadkach `0.0.1`, dlatego zostawiłem tylko to, żeby testy przechodziły w jednym i drugim przypadku. Ale teraz tak sobie myślę, że dołożę to `0.1.0`, tak, żeby testy failowały i przechodziły dopiero kiedy faktycznie zostanie zrobiony release - bo i tak dopiero wtedy to ma być domergowane